js/main.js

```diff
@@ -0,0 +1,281 @@
+const $ul = document.querySelector('ul');
+const $modal = document.querySelector('#modal-section');
+const $class = document.querySelector('#classes-box');
+const $classBox = document.querySelectorAll('.class-box');
+
+$ul.addEventListener('click', handleOperator);
+$class.addEventListener('click', handleFilter);
+
+const xhr = new XMLHttpRequest();
+xhr.open(
+  'GET',
+  'https://lfz-cors.herokuapp.com/?url=https://api.rhodesapi.com/api/operator?exclude=talents,rarity,artist,va,description,quote,voicelines,alter,affiliation,tags,range,statistics,trait,potential,trust,skills,costs,module,base,headhunting,recruitable,availability,release_dates',
+);
+xhr.responseType = 'json';
+xhr.addEventListener('load', function () {
+  console.log(xhr.status);
+  console.log(xhr.response);
+  for (const ops of xhr.response) {
+    if (ops.name !== 'Kirin X Yato') {
+      if (ops.art.length > 1) {
+        createList(ops);
+      }
+    }
+  }
+});
+xhr.send();
+
+function createList(ops) {
+  art = ops.art[1].link;
+  const newImg = document.createElement('img');
+  const newLi = document.createElement('li');
+  newLi.setAttribute('data-id', ops.name);
+  newLi.setAttribute('data-class', ops.class[0]);
+  newImg.classList.add('ops');
+  newImg.setAttribute('src', art);
+  newImg.setAttribute('alt', ops.name);
+  $ul.appendChild(newLi);
```

Comment:
You should be appending to the DOM outside the function. You should be `return`ing the outtermost element of this function, and then wherever you call this function, that's where you would actually append to the DOM